src/app/customer/Customer.tsx

```diff
@@ -52,6 +55,7 @@ export interface WithCheckoutCustomerProps {
     clearError(error: Error): Promise<CheckoutSelectors>;
     continueAsGuest(credentials: GuestCredentials): Promise<CheckoutSelectors>;
     deinitializeCustomer(options: CustomerRequestOptions): Promise<CheckoutSelectors>;
+    executePaymentProviderCheckout(options: ExecutePaymentProviderCheckoutOptions): Promise<CheckoutSelectors>;
```

Comment:
I believe we use the term "payment method" rather than "payment provider" in checkout.

Reply:
Ok, I will change it